cfme/configure/tasks.py

```diff
@@ -221,29 +45,26 @@ class tabs(View):  # noqa
         # It's the lowest level div with an id that captures the status checkboxes
         status = CheckboxSelect(search_root='tasks_options_div')
         state = BootstrapSelect(id='state_choice')
+        table = Table('//div[@id="gtl_div"]//table')
```

Comment:
I question moving this widget, since it is displayed within the tabs defined below. It looks like code duplication, and kind of is, but its driven by the structure of the page. I think it should stay within the tabs.